### ISSUE TITLE
Feature: beautify POST response

### ISSUE THREAD

**Issue by 678098** [Rankuli-Ang/bottle_bot#4]
* For a GET response `/coordinates/` json is being sent. You should send a json for a POST request too then to make it in a common way. In other words, it's somehow ugly to send json in one case and just string in other case.
* `"Bot is already moves"` is not a good description (it represents a matter of fact, like, bot is able to move, but not realtime action). Better use something like `bot is already moving`.
* There is an inconsistency with capital and small letters for response: `Bot` but `movement`. Make sure to make it similar.

So in summary not this:
```
Bot is already moves
```

```
movement is over
```

But this:

```
{
  "status": "movement is over"
}
```

```
{
  "status": "bot is moving"
}
```
